src/clj/harja/kyselyt/paallystys.sql

```diff
@@ -66,41 +73,95 @@ FROM yllapitokohde ypk
 WHERE ypk.id = :paallystyskohde
       AND ypk.poistettu IS NOT TRUE;
 
--- name: paivita-paallystysilmoitus!
--- Päivittää päällystysilmoituksen
+-- name: hae-paallystysilmoitus-paallystyskohteella
+SELECT
+  id,
+  muutoshinta,
+  tila,
+  aloituspvm,
+  valmispvm_kohde                 AS "valmispvm-kohde",
+  valmispvm_paallystys            AS "valmispvm-paallystys",
+  takuupvm,
+  ilmoitustiedot,
+  paatos_tekninen_osa             AS "paatos-tekninen-osa",
+  paatos_taloudellinen_osa        AS "paatos-taloudellinen-osa",
+  perustelu_tekninen_osa          AS "perustelu-tekninen-osa",
+  perustelu_taloudellinen_osa     AS "perustelu-taloudellinen-osa",
+  kasittelyaika_tekninen_osa      AS "kasittelyaika-tekninen-osa",
+  kasittelyaika_taloudellinen_osa AS "kasittelyaika-taloudellinen-osa",
+  asiatarkastus_pvm               AS "asiatarkastus-tarkastusaika",
+  asiatarkastus_tarkastaja        AS "asiatarkastus-tarkastaja",
+  asiatarkastus_tekninen_osa      AS "asiatarkastus-tekninen-osa",
+  asiatarkastus_taloudellinen_osa AS "asiatarkastus-taloudellinen-osa",
+  asiatarkastus_lisatiedot        AS "asiatarkastus-lisatiedot"
```

Comment:
Mielestäni tässä on parempi tehdä näin koska arvot saadaan näkymään suoraan lomakkeella ilman että pitää lukea erillisestä mapista.